server/test/api/debtorGroups.js

```diff
@@ -269,6 +286,18 @@ describe('The /debtor_groups HTTP API ENDPOINT', function () {
       .catch(helpers.handler);
   });
 
+  it('PUT /debtor_groups/ update a debtor group', function () {
```

Comment:
Very nice test! It would be best to write `PUT /debtor_groups/:uuid`, since that is the request you are sending.  It helps when debugging strange errors.